services/web/server/src/simcore_service_webserver/folders/errors.py

```diff
@@ -0,0 +1,20 @@
+from ..errors import WebServerBaseError
+
+
+class FoldersValueError(WebServerBaseError, ValueError):
+    ...
+
+
+class FolderNotFoundError(FoldersValueError):
+    msg_template = "Folder not found. {reason}"
+
+
+class FolderAccessForbiddenError(FoldersValueError):
+    msg_template = "Folder access forbidden. {reason}"
+
+
+# Folder groups
```

Comment:
MINOR: No need for these comments. The prefix is already clear and these comments they end up on the top when reformatting